src/main/java/mate/academy/connection/ConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.connection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/academy?serverTimezone=UTC";
```

Comment:
The database URL seems to be incomplete or broken into two lines. Ensure the DB_URL string is a complete and valid JDBC URL.